examples/shell/cmd_device.cpp

```diff
@@ -0,0 +1,374 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <core/CHIPCore.h>
+
+#if CONFIG_DEVICE_LAYER
+
+#include <platform/CHIPDeviceLayer.h>
+#include <shell/shell.h>
+#include <support/Base64.h>
+#include <support/CHIPArgParser.hpp>
+#include <support/CodeUtils.h>
+#include <support/RandUtils.h>
+
+#include <inttypes.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+using namespace chip;
+using namespace chip::Shell;
+using namespace chip::DeviceLayer;
+using namespace chip::Logging;
+using namespace chip::ArgParser;
+
+#define THE_SHELL_SUBMODULE theShellDevice
+static chip::Shell::Shell THE_SHELL_SUBMODULE;
```

Comment:
is this convention in another project?

Reply:
No, I introduced it to minimize the # of lines that need to be updated when cloning a sub-command module.  I can simplify to just explicitly referencing `theShellDevice` static in the three scattered places in with this file.

Reply:
yeah, would prefer that...  the naming convention used elsewhere in CHIP is `sShellDevice` for such statics

Reply:
Updated names to follow `sStatic` convention.